### DISCUSSION TITLE
AWS S3 Docs

### DISCUSSION THREAD

**Discussion by MohamedXi** [NarHakobyan/awesome-nest-boilerplate#300] [Q&A]
Hello;

I think the idea of using S3 for the management of file downloads is great.

But, I wonder if you have a documentation for the AWS S3 part?
Strangely, it's not my credentials configured in the project with ConfigService that are used but those locally on my host machine.

This is not normal, do we agree?

**Comment by NarHakobyan** [comment]
Het @MohamedXi, Probably you have not configured your AWS configs, you shouls add AWS cli and run `aws configure` to do so
